create_derivatives/routines.py

```diff
@@ -6,7 +9,7 @@
 from pictor import settings
 
 from .clients import ArchivesSpaceClient
-from .helpers import check_dir_exists
+from .helpers import check_dir_exists, matching_files
```

Comment:
This comment cuts across a couple of PRs, but I'm wondering how necessary it is to use the `matching_files` helper in this implementation if we are putting things in subdirectories of bags?

Reply:
Even with the subdirectories of bags, I found matching_files super useful in prepending directory paths (and why I imported it after tests failed because of filepath issues). That helper does a lot so it may be worth seeing which options we end up using across services?

Reply:
Hmm okay, I think it might be good to look at this once we have more things in place. You're right - that helper is doing a lot of work and I'm wondering if we're introducing unnecessary complexity...

Reply:
A note that I did remove the `_001` related functionality when changing `os` to `pathlib`, since I felt confident we won't be using that in this app, but otherwise left things alone for now.